Apps/WebClient/src/ClientApp/src/components/timelineLoading.vue

```diff
@@ -172,13 +172,13 @@ export default class TimelineLoadingComponent extends Vue {
     }
 
     private resetAnimation(elementId: string): boolean {
-        var el: HTMLElement = document.getElementById(elementId);
+        var el: HTMLElement | null = document.getElementById(elementId);
```

Comment:
if you know the element exist for sure, cast it instead of making the type  nullable

Reply:
It could actually be nullable